addons/payment/views/payment_templates.xml

```diff
@@ -166,6 +168,7 @@
               t-att-data-reference-prefix="reference_prefix"
               t-att-data-partner-id="partner_id"
               t-att-data-access-token="access_token"
+              t-att-data-invoice-id="invoice_id"
```

Comment:
```suggestion
```
Not for the manage form.

Reply:
In sale, the sale_order_id is given for the checkout and the manage forms, that's why I specified it in both templates, but indeed, it's not useful ^^.